apps/SageAccountWeb/src/App.scss

```diff
@@ -39,20 +39,17 @@
 }
 
 .SourceAppLogo {
-  img {
-    max-width: 320px;
+  & > .SourceAppImage {
+    height: 65px;
   }
-  svg {
-    height: 100%;
-  }
-  margin-bottom: 50px;
+  margin-bottom: 30px;
   display: flex;
   align-items: center;
   justify-content: center;
 }
 
 .AccountSettingsSourceAppLogo {
-  img {
+  svg {
```

Comment:
Fix sizing of logo in account settings, logo changed to svg in this [commit](https://github.com/Sage-Bionetworks/synapse-web-monorepo/commit/ee3810b058f2dced89e7a86b1aa3173c0880ec2f)